### DISCUSSION TITLE
Custom Containers - title and style

### DISCUSSION THREAD

**Discussion by goozoon** [vuejs/vitepress#1664] [Ideas]
Hi all,
Currently, we have predefined TIP, DANGER (which is not in BOLD, by the way, anymore.. at least in Alpha 29) or we can add our Tittle.
Please can we please have

- an option with no Tittle at all?
- 5th 'custom' container in Grey
- the simplified ability to make a custom design, perhaps like this

![custom-messages](https://user-images.githubusercontent.com/48757340/205292420-aeb39dcd-80a6-4e36-bcbd-05c84afb1c4f.png)

I mean, less is sometimes more. 
Thank you

**Comment by goozoon** [comment]
> (which is not in BOLD, by the way, anymore.. at least in Alpha 29)

Just found out that BOLD font is not displayed once you change font to Custom. My test web has Open+Sans and i see no bold for H1, H2, H3... nowhere. Changed to Roboto, same.

**Comment by goozoon** [comment]
this might be also be a CSS thing, but.. color wise, why default INFO is Grey?

In most cases there is 'always'

* info - BLUE
* tip, success - GREEN
* warning - YELLOW
* error, danger - red

What do you think @kiaking , please?

**Comment by goozoon** [comment]
Please any update on un-labeled containers
In cases when no Title is needed, please?

**Reply by finkarder** [reply]
For no-title + no-label you can hijack the support for html in titles and use an "empty" html element.

```
::: info <i/>
This is an un-labeled container with no title.
:::
```

This works but the styling is slightly off.

**Comment by goozoon** [comment]
un-labeled containers would be realy nice, with no Title
I can add **& nbsp ;**, but then I have an empty space where Title is intended to be

**Reply by brc-dd** [reply]
You can directly use html like this: https://github.com/vuejs/vitepress/blob/20511006dba516ca8c06ed1dd3516547af668a0e/docs/guide/what-is-vitepress.md?plain=1#L5-L9

rendered:

<img width="785" alt="image" src="https://github.com/vuejs/vitepress/assets/40380293/af2ee2b4-0c80-44cf-80ff-9e31745b642a">

**Reply by goozoon** [reply]
Thank you
As a workaround or permanent solution?
